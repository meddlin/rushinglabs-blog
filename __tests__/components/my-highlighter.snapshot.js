import { render } from '@testing-library/react';
import MyHighlighter from '../../components/content-helpers/my-highlighter';

it('renders MyHighlighter unchanged', () => {
    const { container } = render(
        <MyHighlighter
            language="js"
            showLineNumbers={true}
            showInlineLineNumbers={true}
            >
                this is a test
        </MyHighlighter>
    );

    expect(container).toMatchSnapshot()
});