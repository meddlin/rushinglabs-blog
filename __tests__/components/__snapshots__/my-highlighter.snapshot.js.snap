// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MyHighlighter unchanged 1`] = `
<div>
  <pre
    class="text-xs"
    style="display: block; overflow-x: auto; background: rgb(43, 43, 43); color: rgb(248, 248, 242); padding: 0.5em;"
  >
    <code
      class="language-js"
      style="white-space: pre;"
    >
      <span
        class="comment linenumber react-syntax-highlighter-line-number"
        style="display: inline-block; min-width: 1.25em; padding-right: 1em; text-align: right; user-select: none;"
      >
        1
      </span>
      <span
        style="color: rgb(220, 198, 224);"
      >
        this
      </span>
      <span>
         
      </span>
      <span
        style="color: rgb(220, 198, 224);"
      >
        is
      </span>
      <span>
         a test
      </span>
    </code>
  </pre>
</div>
`;
